### DISCUSSION TITLE
xuexiluxian/go

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#265] [Announcements]
# xuexiluxian/go

Go 语言越来越火了，这里整理一份 go 语言的超详细学习路线，附学习资源，可下载

https://javabetter.cn/xuexiluxian/go.html

**Comment by xiaochuan201** [comment]
写作日期怎么是未来的日期。【2026年7月17日】

**Comment by cong1ling** [comment]
这是在发表未来的文章吗
